blocks/subscriptions-block/features/checkout/default.jsx

```diff
@@ -1,27 +1,40 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, useMemo } from "react";
 import PropTypes from "@arc-fusion/prop-types";
 import { usePhrases, Heading, Link, useIdentity } from "@wpmedia/arc-themes-components";
-import useSales from "../../components/useSales";
+import usePaymentOptions from "../../components/usePaymentOptions";
 import Cart from "../../components/Cart";
 import ContactInfo from "../../components/ContactInfo";
 import PaymentInfo from "./_children/PaymentInfo";
 
 const BLOCK_CLASS_NAME = "b-checkout";
 
```

Comment:
Since LABEL_ORDER_NUMBER_PAYPAL it's a constant, maybe define here at the beginig